db/schema.rb

```diff
@@ -10,17 +10,27 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-
-ActiveRecord::Schema.define(version: 20200512205827) do
-
-# ActiveRecord::Schema.define(version: 20200512020737) do
-# This was a merge conflict but wasnt sure if deleting it was the best option. Otherwise version above is from mine
-
+ActiveRecord::Schema.define(version: 20200513215451) do
 
```

Comment:
seems like everytime we make a migration were going to have to deal with one of these conflicts

Reply:
Yeah Im assuming we use the most recent one.